extensions/wikia/VideoHandlers/VideoFileUploader.class.php

```diff
@@ -159,7 +159,17 @@ public function upload( &$oTitle ) {
 			$file->forceMetadata( serialize( $this->getNormalizedMetadata() ) );
 		}
 
+
+		$forceMime = $file->forceMime;
+
 		$file->getMetadata();
+
+		//In case of video replacement - Title already exists - preserve forceMime value
+		//By default it is changed to false in WikiaLocalFileShared::afterSetProps method
```

Comment:
Can you specify that WikiaLocalFileShared::afterSetProps is called by `$file->getMetadata();` above? At first glance it looks like we're just reassigning `$file->forceMime` to itself.